Package.swift

```diff
@@ -1,11 +1,11 @@
-// swift-tools-version: 5.5
+// swift-tools-version: 5.7
```

Comment:
bumped this so [v16](https://developer.apple.com/documentation/packagedescription/supportedplatform/iosversion/v16) was available